package.json

```diff
@@ -104,7 +105,7 @@
     "parse-svg-path": "^0.1.2",
     "polybooljs": "^1.2.0",
     "probe-image-size": "^7.2.2",
-    "regl": "^2.1.0",
+    "regl": "git@github.com:yujin-wu/regl.git#plotly/cache-codegen",
```

Comment:
Let's try uninstalling
```sh
npm uninstall regl
```
and then install the hash
```sh
npm install --save git://github.com/yujin-wu/regl.git#29c9b09ca5f078b674131e20f3a99197550521b5
```
Please commit changes to `package.json` and `package-lock.json`.